packages/react-resizable-panels/src/index.ts

```diff
@@ -5,17 +5,27 @@ import { PanelResizeHandle } from "./PanelResizeHandle";
 import type { ImperativePanelHandle, PanelProps } from "./Panel";
 import type { ImperativePanelGroupHandle, PanelGroupProps } from "./PanelGroup";
 import type { PanelResizeHandleProps } from "./PanelResizeHandle";
-import type { PanelGroupStorage } from "./types";
+import type {
+  PanelGroupOnLayout,
+  PanelGroupStorage,
+  PanelOnCollapse,
+  PanelOnResize,
+  PanelResizeHandleOnDragging,
+} from "./types";
 
 export {
   // TypeScript types
   ImperativePanelGroupHandle,
   ImperativePanelHandle,
   Panel,
+  PanelOnCollapse,
```

Comment:
Those added things were not exported but yet they were consumable... I have no idea how that worked. If we inspect the content od the current declaration files we can clearly see that the types that I had to export from here were *not* exported:
https://unpkg.com/browse/react-resizable-panels@0.0.45/dist/react-resizable-panels.d.ts

Reply:
I found out why this worked before (with @emmatown's help) and opened a PR in Parcel with a change that would help to avoid accidental leakage like this: https://github.com/parcel-bundler/parcel/pull/9016